client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.scss

```diff
@@ -0,0 +1,40 @@
+.edit-post-pinned-plugins,
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			width: 20px;
+			height: 20px;
+		}
+	}
+}
+
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			margin-right: 4px;
+		}
+	}
+}
+
+.edit-post-pinned-plugins {
+	.components-icon-button {
+		&:not( .is-toggled ),
+		&:hover,
+		&.is-toggled,
+		&.is-toggled:hover {
+			.jetpack-logo,
+			.jetpack-logo * {
+				fill: none !important;
```

Comment:
While it's generally bad to have `!important` in any styles, this is required, as otherwise Gutenberg will override the colors inside our SVG.

Reply:
The entire selector is kinda awful :stuck_out_tongue_closed_eyes: 

Specifically, do we need this `fill` override? In my element inspector, it looks like it's being overridden by https://github.com/Automattic/wp-calypso/pull/28913/files#diff-b3f6b257d3217c67ad46a47c0acf88bbR32 anyway.

Reply:
I've landed https://github.com/Automattic/wp-calypso/pull/28920 so I expect we can drop this `*` selector bit at least.

Reply:
I've removed the unnecessary `fill` in 82aec101c8a1e3d92e0ab3da06b75905211ea988 and removed the `*` selector in favor of concrete selectors in 3d529479713f76bb4228c63fd8ce5fe99a767a72 - we still need to remove the stroke that Gutenberg adds.